seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1133,8 +1133,9 @@
       ],
       "tests": [
         "assert(!/\\\\/g.test(code) && myStr === '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>', 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
-        "assert(code.match(/\"/g).length === 4 && code.match(/'/g).length === 2, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
+        "assert((function() { const strict = \"'use strict';\"; const tail = '(function() { return \"myStr = \" + myStr; })();'; const sliced = code.slice(code.indexOf(strict) + strict.length, code.indexOf(tail)); if (sliced.match(/\"/g).length === 4 && sliced.match(/'/g).length === 2) return true; return false; })() === true, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
```

Comment:
@Greenheart good idea - yeah I thought of this that the test would need to be changed if the tail ever changes, so this sounds like a safer bet, I should've thought of this myself 😄